.github/workflows/main.yaml

```diff
@@ -65,6 +65,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Update submodule
```

Comment:
checkout action support with a `submodules`  parameter to enable submodule checkout. seems like:
```
- name: checkout
  uses: actions/checkout@v2
  with:
     submodules: 'true'
```

Reply:
Seems submodules: 'true' only support SSH mode.